### PR TITLE
Blog posts on load

### PR DESCRIPTION
**Pull request by Glombort**
Added blog posts on load, with object for posts to be taken from. Add your own messages too for load :)

### CODE REVIEW

File: blogPosts.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let posts = {
+    0: {user: 'George', message: 'Alligator &gt Crocodile'},
+    1: {user: 'Asmahan', message: ''},
+    2: {user: 'Maria', message: ''},
+    3: {user: 'Bereket', message: ''},
+    4: {user: 'Elon', message: 'Damn this is better than twitter. Can I buy? &#128176;'},
```
Comment (mariaalouisaa) [inline blogPosts.js:6]: We're gonna get rich from this, ey 😅

File: blogPosts.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+};
+
+module.exports = posts;
```
Comment (mariaalouisaa) [inline blogPosts.js:9]: Yaay, glad you went ahead and did it like this!! I feel like we're off to a good start.

File: routes/home.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+const blogPosts = require("../blogPosts.js");
+
+function get(request, response) {
+    let posts = '';
+    for (const post of Object.values(blogPosts)) {
+        posts += `
+        <div class="blog-post"><h3>${post.user}</h3> <p>${post.message}</p></div>
```
Comment (mariaalouisaa) [inline routes/home.js:7]: Is this where we would implement the hack-proofing? 👀 Though not sure how much damage people can do with a username and message...

File: routes/home.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        <div>
+            <header>
+                <h2>Posts</h2>
+            </header>
+            ${posts}
+        </div>
+      </body>
+    </html>
+    `
+    response.send(html)
```
Comment (mariaalouisaa) [inline routes/home.js:27]: Nice you put the HTML into a variable, code is looking tidy and very readable.

File: server.js [hunk 5]
```diff
@@ -1,9 +1,9 @@
 const express = require('express');
+const home = require("./routes/home.js");
+
 const server = express();
 
-server.get('/', (request, response) => {
-   response.send('hello')
-})
+server.get('/', home.get)
```
Comment (mariaalouisaa) [inline server.js:5]: I had to process this a couple of times but I think I see how this works, so...
1) The get function is being exported from the home.js file and then required (imported) to the server.js file
2) The contents of the get function are being "served" by the server.js so is then displayed on the page http://localhost:3000/

Maybe it's a little confusing having the function called "get" cos there are a few gets on the page to follow hahaha
But apart from that, the code is smashing!!!

**Review by mariaalouisaa — COMMENTED** [review]
Looks great!! Your code is super nice to read 😊

**Review by mariaalouisaa — COMMENTED** [review]
Looks great!! Your code is super nice to read 😊

File: routes/home.js (hunk 3 above)
Comment (Glombort) [reply routes/home.js:7]: I think the xss protection comes in the form submission post.

**Pull request merged**